feat: LEAP-219: Update LSF E2E with recent changes

hlomzik opened the pull request:
Sync recent changes made to LSF E2E after during transition to monorepo:
- https://github.com/HumanSignal/label-studio-frontend/pull/1192 for Magic Wand from @BradNeuberg
- https://github.com/HumanSignal/label-studio-frontend/pull/1708 with external images problem
- https://github.com/HumanSignal/label-studio-frontend/pull/1021 for maxUsages missed for some reason
- Fix newlines in all LSF E2E from CRLF to LF


### PR fulfills these requirements
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)


### Describe the reason for change
Some changes were missed because of amount of work needed to move the whole repo to monorepo.


### What level of testing was included in the change?
- [x] e2e
- [ ] integration
- [ ] unit

hlomzik committed:
feat: LEAP-219: Update LSF E2E with recent changes

- https://github.com/HumanSignal/label-studio-frontend/pull/1192 for Magic Wand from @BradNeuberg
- https://github.com/HumanSignal/label-studio-frontend/pull/1708 with external images problem
- https://github.com/HumanSignal/label-studio-frontend/pull/1021 for maxUsages missed for some reason

hlomzik committed:
Fix newlines in all LSF E2E from CRLF to LF

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8429166524

bmartel commented on web/libs/editor/tests/e2e/tests/ner-text.test.js:
```diff
@@ -259,3 +259,26 @@ Scenario('NER Text with SECURE MODE', async function({ I, LabelStudio }) {
+  };
+
+  I.amOnPage('/');
+  // enabling both flags for New UI, but will work with Otliner one only as well
```
```suggestion
  // enabling both flags for New UI, but will work with Outliner one only as well
```

bmartel committed:
Update web/libs/editor/tests/e2e/tests/ner-text.test.js

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8429666577

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8453240101)
> Successfully merged:  18 files changed, 141 insertions(+), 97 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-leap-219/update-lsf-e2e'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8453240101
